tools/tests-consolidated/security/environment-validation.test.ts

```
@@ -0,0 +1,512 @@
+      expect(isValidPackageVersion).toBe(true);
+      expect(isValidSupabaseJsVersion).toBe(true);
+      expect(isValidTypesNodeVersion).toBe(true);
+      expect(isValidTypescriptVersion).toBe(true);
+      expect(isValidSupabaseCliVersion).toBe(true);
+      expect(isValidPostgresJsVersion).toBe(true);
+    });
+  });
+});
+
+// Export for use in other test files
+export { EnvironmentSecurityValidator };
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)